apps/backend/src/libs/modules/open-ai/libs/tools/change-task.tool.ts

```diff
@@ -0,0 +1,112 @@
+import { OpenAIFunctionName } from "../enums/open-ai-function-name.enum.js";
```

Comment:
```suggestion
import { OpenAIFunctionName } from "../enums/enums.js";
```

Reply:
I'm not sure why but I have  an error:
`name: OpenAIFunctionName.ANALYZE_BALANCE_SCORES,
                      ^
ReferenceError: Cannot access 'OpenAIFunctionName' before initialization
`

I use OpenAIFunctionName inside the module and there shouldn't be any errors 
so I need help with this => or it's too simple and I don't see my mistake or smth else

Reply:
Ok, let's keep it